src/components/DocLayout/index.js

```diff
@@ -1,8 +1,8 @@
 import React, { useCallback, useState } from 'react'
 import PropTypes from 'prop-types'
+import MainLayout, { LayoutModifiers } from '../MainLayout'
```

Comment:
should we move DocLayout into Document/Layout?

Reply:
Do you want to add some namespaces for docs (and maybe blog) components?

Reply:
we already have a directory `Documentation` - we can move Layout for it there, right?

if do so we don't need Doc prefix

Reply:
Moved it to `Documentation` directory